### PR TITLE
Add support for chart resources

### PR DESCRIPTION
**Pull request by sbender9**
Allows freeboard-sk to load charts

### CODE REVIEW

File: lib/interfaces/charts.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  var chartFiles = [];
+
+  var host = req.headers.host
+  
+  var httpProtocol = 'http://';
+  if (app.config.settings.ssl) {
+    httpProtocol = 'https://';
+  }
+
+  var url = httpProtocol + host + '/mapcache/'
```
Comment (tkurki) [inline lib/interfaces/charts.js]: Can't you do without the protocol and host here?

File: lib/interfaces/charts.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017 Teppo Kurki <teppo.kurki@iki.fi>
```
Comment (tkurki) [inline lib/interfaces/charts.js:2]: Maybe put yourself here?

**Comment by tkurki** [comment]
Add instructions to readme?

**Comment by tkurki** [comment]
Accidental close..

**Comment by tkurki** [comment]
I've been thinking about starting to use https://github.com/prettier/prettier - could you please format the code with that?

File: lib/interfaces/charts.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      {
+        res.status(404).send("Not found");
+      }
+    })
+    
+    app.get(apiPathPrefix + "resources/charts", (req, res, next) => {
+      var charts = loadCharts(app, req)
+      if ( typeof charts === 'undefined' )
+      {
+        res.json({})
```
Comment (tkurki) [inline lib/interfaces/charts.js:51]: Not very useful response.

Comment (sbender9) [reply lib/interfaces/charts.js:51]: I'm not sure. Should it be a 404 instead? There are no charts.

File: lib/interfaces/charts.js (hunk 1 above)
Comment (sbender9) [reply lib/interfaces/charts.js]: I coped this code from your stuff, need to review...

File: lib/interfaces/charts.js (hunk 3 above)
Comment (sbender9) [reply lib/interfaces/charts.js:51]: I Love you man, but I'm going to format code the way I format code. Feel free to fix it.

**Comment by tkurki** [comment]
Reformatted and squashed as 9ee379a084cba706f3398089d9b34644ee1b7a7a and merged in 281342a681b27956f5c62850555b5eae9ef45c38.

**Pull request closed without merging**